TemplateApplication/TemplateApplicationStandard.swift

```diff
@@ -62,9 +61,17 @@ actor TemplateApplicationStandard: Standard,
     }
 
 
-    init() {
-        if !FeatureFlags.disableFirebase {
-            _accountStorage = Dependency(wrappedValue: FirestoreAccountStorage(storeIn: TemplateApplicationStandard.userCollection))
+    init() {}
+
+
+    private func userDocumentReference(for accountId: String) -> DocumentReference {
+        Self.userCollection.document(accountId)
+    }
```

Comment:
Similar to ENGAGE-HF, we should move that into a dedicated extension on the Firestone type or into a dedicated module that has access to the Account and therefore ID?